app/events/accounts/account_deleted_event.rb

```diff
@@ -5,7 +5,6 @@ def self.create(account)
       account_name: account.name,
       provider:     account.provider_account,
       buyer:        account.buyer,
-      user_id:      account.first_admin.try(:id) || account.first_admin_id_on_account_signup,
```

Comment:
@Martouta could you explain why we no longer need this?

Reply:
Sure @thomasmaas 👍 What I will explain in this comment applies to the entire 3rd commit 😄

The `AccountDeletedEvent` right now in the master branch is used for sending emails and currently for updating segment as `state: deleted` of the admin user of the account that has been deleted. For emails, the `user_id` is not needed, this is something that I discovered for [a bug that we had in November that sometimes the user_id was nil because the user was already deleted and we didn't have the first_admin_id back then](https://github.com/3scale/porta/issues/10#issuecomment-441031162).

Back then to fix that bug what I did was [adding the column first_admin_id in Account](https://github.com/3scale/porta/pull/8), but it was just a bugfix of the invalid and unsaved `AccountDeletedEvent`.

Now what we want is not notifying segment for deleted accounts but for deleted users (and when an account is deleted, all its users are deleted), and we also want to delete for real and not just mark the state deleted 😄

Reply:
Ah now I understand ... Please ignore my comments before (time wise) this one.
We do not need first_admin_id to send email but for Segment. So now that the Segment bug is fixed we can remove it. 👍